### ISSUE TITLE
'Lock' menu option doesn't work

### ISSUE THREAD

**Issue by xfbs** [elementary/wingpanel-indicator-session#50]
On my machine, hitting the 'lock' item in the session indicator doesn't do anything. Manually locking with `dm-tool lock` works.

**Comment by btkostner** [comment]
Can we get some system information? What version of elementary are you running? What version of `wingpanel-indicator-session` are you running?

**Comment by btkostner** [comment]
Seems I am encountering this as well on elementary Juno alpha. `wingpanel-indicator-session` version `2.0.4+r249+pkg12~daily~ubuntu5.0.1`. It's also worth mentioning that the lock keyboard shortcut does not work.

**Comment by mdh34** [comment]
This is a duplicate of elementary/greeter#92, pending on a patched GSD
